crates/re_space_view_time_series/src/view_part_system.rs

```diff
@@ -171,6 +185,12 @@ impl TimeSeriesSystem {
                 continue;
             }
 
+            let min_time = store
+                .entity_min_time(&query.timeline, ent_path)
+                .map_or(points[0].time, |time| time.as_i64()); // point is not empty
```

Comment:
`points` isn't empty until someone moves the code the code around in the future...
```suggestion
                .map_or(points.first().map(|p| p.time).unwrap_or(0), |time| time.as_i64());